plugins/lookup/dsv.py

```diff
@@ -112,7 +112,7 @@ def run(self, terms, variables, **kwargs):
         self.set_options(var_options=variables, direct=kwargs)
 
         vault = LookupModule.Client(
-            **{
```

Comment:
Good catch! I think this one actually needs a changelog fragment, since it probably made the lookup plugin unusable. CC @amigus